src/reporter.js

```diff
@@ -56,10 +61,24 @@ function loadConfiguration(options) {
   config.videosFolder = path.normalize(objConfig.resolved.videosFolder.value);
   config.screenshotsFolder = path.normalize(objConfig.resolved.screenshotsFolder.value);
 
+  // https://github.com/archfz/cypress-terminal-report#log-specs-in-separate-files
+  if (objConfig.logsOptions && 'outputRoot' in objConfig.logsOptions && 'outputTarget' in objConfig.logsOptions) {
+    var supportType = getKeyByValue(objConfig.logsOptions.outputTarget, 'txt');
+    if (supportType !== undefined) {
+      let output = supportType.split('|')
+      config.logFileExt = "."+output[1];
+      config.logsFolder = path.join(path.normalize(objConfig.logsOptions.outputRoot), output[0]);
+    }
+  }
+
+  if (objConfig.logsOptions && config.logsFolder === null) {
+    console.log("Terminal Report Configuration for Reporter not valid:", objConfig.logsOptions)
+  }
```

Comment:
That is incorrect! We should not be printing errors for "differently" configured terminal.